backend/src/apps/owasp/api/internal/queries/chapter.py

```
@@ -38,3 +42,27 @@ def recent_chapters(self, limit: int = 8) -> list[ChapterNode]:
+        if (
+            len(cleaned_query) < MIN_SEARCH_QUERY_LENGTH
+            or len(cleaned_query) > MAX_SEARCH_QUERY_LENGTH
+        ):
+            return []
+
+        return Chapter.active_chapters.filter(
+            name__icontains=cleaned_query,
+        ).order_by("name")[:SEARCH_CHAPTERS_LIMIT]
+
+    @strawberry_django.field
+    def is_chapter_leader(self, info: strawberry.Info, login: str) -> bool:
```

Ruff (0.16.1)

[warning] Unused method argument: `info`

(ARG002)